dev/Gems/LmbrCentral/Code/Source/Scripting/TriggerAreaComponent.cpp

```diff
@@ -353,6 +356,30 @@ namespace LmbrCentral
         }
     }
 
+	void TriggerAreaComponent::AddIgnoredEntity(const AZ::EntityId& ignoredEntityId)
+	{
+		const auto& ignoredEntityIter = AZStd::find(m_ignoredEntities.begin(), m_ignoredEntities.end(), ignoredEntityId);
```

Comment:
This returns an iterator type by value, so it should just be auto.

Also why not make m_ignoredEntities an unordered_set?

Reply:
m_ignoredEntities is just following in the footsteps of the existing code for example `AZStd::vector<AZ::EntityId>     m_entitiesInsideExcludedByTags;` and `AZStd::vector<AZ::EntityId>     m_entitiesInside;`

I considered using a typedef and changing all of the existing containers to something else but reasoned that I would:
a) Make the smallest change possible.
b) Not change existing already tested code.
I know it isn't a scary change but it wasn't necessary here.